BundID/Views/Setup/SetupTransportPIN.swift

```diff
@@ -20,70 +20,73 @@ let setupTransportPINReducer = Reducer<SetupTransportPINState, SetupTransportPIN
 
 struct SetupTransportPIN: View {
     
-    var store: Store<SetupTransportPINState, SetupTransportPINAction>
+    let store: Store<SetupTransportPINState, SetupTransportPINAction>
     @State var digits = 5
     
     init(store: Store<SetupTransportPINState, SetupTransportPINAction>) {
         self.store = store
     }
     
     var body: some View {
-        VStack(alignment: .leading) {
-            ScrollView {
-                VStack(alignment: .leading, spacing: 24) {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 24) {
+                Text(L10n.FirstTimeUser.TransportPIN.title)
+                    .font(.bundLargeTitle)
+                    .foregroundColor(.blackish)
+                Text(L10n.FirstTimeUser.TransportPIN.body)
+                    .font(.bundBody)
+                    .foregroundColor(.blackish)
+                ZStack {
+                    Asset.transportPIN.swiftUIImage
```

Comment:
Das sorgt dafür, dass das Bild ein Label bekommt (für Accessibility), was wir vorher explizit ausgeschaltet haben (`decorative`-init).

```suggestion
                    Image(decorative: Asset.transportPIN)
```